l10n/en/mozorg/about/this-site.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/about/this-site/
+
+about-this-site-title = About this site
+about-this-site-desc = About the website www.mozilla.org and the open source tools used to make it.
+
+# Variables:
+#   $archive (url) link to https://web.archive.org/web/19981212031129/http://www.mozilla.org/
+#   $coderush (url) link to https://www.youtube.com/watch?v=4Q7FTjhvZ7Y
+about-this-site-this-website = The website www.mozilla.org has been online for over two decades. That’s since <a href="{ $archive }">dinosaur times</a> in internet years. As such, www.mozilla.org is a corner stone of the web, with roots embedded in the open source movement that saw the <a href="{ $coderush }">creation of { -brand-name-mozilla }</a>.
+
+# Variables:
+#   $firefox (url) link to https://www.mozilla.org/firefox/new/
+#   $vpn (url) link to https://www.mozilla.org/products/vpn/
+#   $mozilla (url) link to https://www.mozilla.org/about/
+#   $book (url) link to https://www.mozilla.org/book/
+about-this-site-today = Today, this is the site where people come to download <a href="{ $firefox }">{ -brand-name-firefox }</a>, try <a href="{ $vpn }">{ -brand-name-mozilla-vpn }</a>, and learn more <a href="{ $mozilla }">about { -brand-name-mozilla }</a>. You can also discover a few <a href="{ $book }">easter eggs</a> along the way.
+
+about-this-site-like-many = Like many of our products, this website is also open source:
+about-this-site-view-source = View the source code on { -brand-name-github }
+about-this-site-report-a-bug = Report a bug
+about-this-site-contribute-to-code = Contribute to the codebase
+about-this-site-contribute-to-l10n = Contribute to translations
+about-this-site-read-the-docs = Read the documentation
+about-this-site-colophon = Colophon
+about-this-site-some-open-source-projects = Some open source projects used in making this site:
+
+# Variables:
+#   $django (url) link to https://www.djangoproject.com/
+#   $jinja (url) link to https://jinja.palletsprojects.com/
+about-this-site-django = <a href="{ $django }">Django</a> as a back end web framework, with <a href="{ $jinja }">Jinja</a> for templating.
+
+# Variables:
+#   $protocol (url) link to https://protocol.mozilla.org/
+about-this-site-mozilla-protocol = { -brand-name-mozilla } <a href="{ $protocol }">Protocol</a> design system for front end components and branding.
+
+# Variables:
+#   $fluent (url) link to https://projectfluent.org/
+about-this-site-fluent = { -brand-name-mozilla } <a href="{ $fluent }">Fluent</a> localization system for language translation.
```

Comment:
Since Fluent isn't titled `Mozilla Fluent`, we should probably add an apostrophe after `Mozilla` here, indicating that it's a project by Mozilla and that it's titled `Fluent` (or `Project Fluent`).
```suggestion
about-this-site-fluent = { -brand-name-mozilla }’s <a href="{ $fluent }">Fluent</a> localization system for language translation.
```